packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -24,17 +33,17 @@ export default {
     {
       type: 'slot',
       name: 'default',
-      value: `<p class="sd-paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed.</p>`
+      value: `<div class="slot slot--border slot--text h-16 w-full">Default slot</div>`
     },
     {
       type: 'slot',
       name: 'headline',
-      value: `<h4 slot="headline" class="sd-headline sd-headline--size-3xl leading-tight">Lorem Ipsum</h4>`
+      value: `<div slot="headline" class="slot slot--border slot--text h-16">Headline slot</div>`
     },
     {
       type: 'slot',
       name: 'footer',
-      value: `<sd-button slot="footer" class="w-full">Label</sd-button>`
+      value: `<div slot="footer" class="slot slot--border slot--text h-16 w-full">Footer slot</div>`
```

Comment:
Please change the with of footer like in figma: 
<img width="966" alt="image" src="https://github.com/user-attachments/assets/ffe82682-b247-48c3-afd0-53f4a9a48ea0">

Reply:
The footer slot is right aligned, therefore in my opinion the update should be done in Figma. 
FYI @coraliefeil

Reply:
After a chat with @mariohamann we noticed that there is bug in the footer styling. I've created a [ticket](https://github.com/solid-design-system/solid/issues/1325) to address it.
@coraliefeil we can keep figma as it is and will update storybook once the bugfix is addressed. 
FYI @yoezlem